R/data.R

```diff
@@ -1,9 +1,13 @@
 # Tributaries and Mainstems -----
 #' Flow in cfs
-#' @description A dataset containing the flow in cubic feet per second within watersheds for
+#' @description A named list containing datasets from the 2008-2009 and 2018-2019 Calsim II BiOp runs for
 #' use with the CVPIA SIT Salmon Population Model to develop habitat inputs.
+#' The named list components are:
```

Comment:
Did you check that the formatting rendered as expected here? I think that with oxygen you may need to use \itemize{\item{}} instead of bullets